examples/MQ/pixelDetector/src/PixelDigiBinSource.h

```diff
@@ -28,12 +28,10 @@ class PixelDigiBinSource : public FairSource
 {
   public:
     PixelDigiBinSource(TString inputFileName = "test.dat");
-    ~PixelDigiBinSource() override = default;
```

Comment:
Should we follow "the rule of zero"? Here is a discussion about the rule of zero for derived classes: https://stackoverflow.com/questions/21637250/does-the-rule-of-zero-also-apply-for-classes-with-virtual-methods

Reply:
The second comment has an interesting part:

> `= default`ing special members does not violate the Ro0 in any way.

That's part one of my (our?) take on this.

Part two goes like this:
- The base class of a class with virtual methods *should* have a virtual dtor.
- How can we make sure in a derived class, that the base class has a virtual dtor? By using `override`!
- As we don't want to implement any special logic (Ro0?), let's `= default` it.

Reply:
I think except the constructor, we make special member functions default by not defining it. Otherwise, you should also default 5 member functions. Anyway, this is a stylish issue and I will change it according to your standard.